CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 5.0.0-alpha.0 (2021-06-11)
+
+### New features
+
+- Added support for `@angular/core@^12`. [#223](https://github.com/blackbaud/skyux-core/pull/223)
+- Updated the following singleton services to use [Angular's preferred method to become tree-shakeable](https://angular.io/guide/singleton-services#using-providedin): `SkyCoreAdapterService`, `SkyAffixService`, `SkyDockService`, `SkyDynamicComponentService`, `SkyAppFormat`, `SkyLogService`, `SkyMediaQueryService`, `MutationObserverService`, `SkyNumericService`, `SkyOverlayAdapterService`, `SkyOverlayService`, `SkyAppTitleService`, `SkyUIConfigService`, `SkyViewkeeperService`, and `SkyAppWindowRef`.
```

Comment:
Does this line also require the link to the PR? [#223](https://github.com/blackbaud/skyux-core/pull/223)

Reply:
Good catch. fixed!